docker/jupyter/scripts/extension/jupyter_install_lab_extension.sh

```diff
@@ -5,6 +5,17 @@ set -e
 
 if [ -n "$1" ]; then
   JUPYTER_EXTENSION=$1
-  jupyter labextension install $JUPYTER_EXTENSION
+  if [[ ${JUPYTER_EXTENSION} == *'.js' ]]; then
```

Comment:
I had thought we want to support `*.ts` extensions (and use https://github.com/jupyterlab/extension-cookiecutter-ts). Maybe check with @zarsky-broad -- should we support `.js`, `.ts`, or both?

Reply:
just checked with Isaac, it's js!

Reply:
Sorry to change my mind -- I was thinking maybe we should just support `js` or `ts` since it's easy and will make leo more flexible. So: 
```
if [[ ${JUPYTER_EXTENSION} == *'.js' ]]; then
  <current stuff>
elif [[ ${JUPYTER_EXTENSION} == *'.ts' ]]; then
  JUPYTER_EXTENSION_NAME=`basename ${JUPYTER_EXTENSION%%.*}`
  cookiecutter --no-input https://github.com/jupyterlab/extension-cookiecutter-ts extension_name=${JUPYTER_EXTENSION_NAME}
  cp -f ${JUPYTER_EXTENSION} ${JUPYTER_EXTENSION_NAME}/src/index.ts
  cd ${JUPYTER_EXTENSION_NAME}
  jlpm
  jupyter labextension install .
else 
  <current stuff>
fi
```

I am ok with only having an auto test for `.js` if that's what Terra will be using.

Reply:
Sounds good - just added that!